### PR TITLE
Add tests for debrepo split_pkg_path

### PR DESCRIPTION
**Pull request by knazarov**
This pull request adds basic tests for split_pkg_path to be used in validation of #29

### CODE REVIEW

File: test/test_debrepo.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                'pool/main/t/tarantool/libtarantool-dev_1.5.2.20.g5f5d924-2_amd64.deb',
+                ('all', 'main', 'amd64')
+            ),
+            (
+                'pool/main/v/vim/vim_8.2.2434-3_amd64.deb',
+                ('all', 'main', 'amd64')
+            ),
+            # Commented out because it fails to parse properly at the moment
+            # The "dist" should be "main", but parses to "1" with current parser
+            #(
```
Comment (LeonidVas) [inline test/test_debrepo.py:23]: Each line of a block comment starts with a # and a single space '# ' (https://www.python.org/dev/peps/pep-0008/#block-comments)

File: test/test_debrepo.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            #    'pool/main/t/tarantool/tarantool-python_0.6.6-21_amd64.deb',
+            #    ('all', 'main', 'amd64')
+            #),
```
Comment (LeonidVas) [inline test/test_debrepo.py:26]: Each line of a block comment starts with a # and a single space '# ' (https://www.python.org/dev/peps/pep-0008/#block-comments)

File: test/test_debrepo.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            # Neither does this
+            #(
```
Comment (LeonidVas) [inline test/test_debrepo.py:28]: Each line of a block comment starts with a # and a single space '# ' (https://www.python.org/dev/peps/pep-0008/#block-comments)

File: test/test_debrepo.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            #    'debian/pool/main/o/openssl/libssl1.1_1.1.0l-1~deb9u1_amd64.deb',
+            #    ('1~deb9u1', 'main', 'amd64')
+            #),
```
Comment (LeonidVas) [inline test/test_debrepo.py:31]: Each line of a block comment starts with a # and a single space '# ' (https://www.python.org/dev/peps/pep-0008/#block-comments)

File: test/test_debrepo.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        ]
+
+        for version in versions:
+            self.assertEqual(
+                debrepo.split_pkg_path(version[0]), version[1],
+                version[0])
```
Comment (LeonidVas) [inline test/test_debrepo.py:37]: I think this message should be more verbose (programmer friendly).

**Review by LeonidVas — CHANGES_REQUESTED** [review]
Hi! Thank you for the patch.
Generally ok but I have several comments.
I also want to suggest that you use https://github.com/tarantool/tarantool/wiki/Code-review-procedure#commit-message for style of commits, because in this case it is easy enough to understand from the commit title/message what was done, where and why.

File: test/test_debrepo.py (hunk 5 above)
Comment (knazarov) [reply test/test_debrepo.py]: I've removed the message completely, because what unittest shows by default is good enough.

**Comment by knazarov** [comment]
@LeonidVas I've updated the code according to comments

**Review by LeonidVas — APPROVED** [review]
LGTM.

**Comment by Totktonada** [comment]
Nice, thank you guys!

**Pull request merged**